MobileFilter/sections/specific_web.txt

```diff
@@ -69,6 +69,7 @@ n-tv.de##.topbanner-mobile
 syosetu.org##.ad
 m.bloomberght.com##.zone-area
 ezgif.com###hdr
+kompas.com###js-banner-sticky
```

Comment:
I don't see this element

Reply:
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/114567106-96cf2380-9c7b-11eb-8715-46d436b99812.png)</details><br/>

Reply:
`kompas.com##.boxBanner1` from AdBlockID hides this element